rake-tasks/crazy_fun/mappings/ruby.rb

```diff
@@ -104,24 +104,20 @@ def handle(_fun, dir, args)
         end
 
         ruby :include => args[:include],
-             :require => args[:require],
              :command => args[:command],
              :args    => %w[--format doc --color] + (!!ENV['example'] ? ['--example', ENV['example']] : []),
              :debug   => !!ENV['log'],
-             :files   => args[:srcs],
-             :gemfile => "build/rb/Gemfile"
```

Comment:
Now we only use `rb/Gemfile` because we need lock file to run commands.

It might be error-prone on CI, but it should not be a big deal.